modules/migration-using-must-gather.adoc

```diff
@@ -39,7 +39,25 @@ endif::[]
 . Navigate to the directory where you want to store the `must-gather` data.
 . Run the `oc adm must-gather` command for one of the following data collection options:
 
-ifdef::troubleshooting-3-4,troubleshooting-mtc[]
+ifdef::troubleshooting-3-4[]
+* To collect data for the past hour:
```

Comment:
This really should be a more complete sentence, such as "To collect data for the past hour, use the following command:" (replicating what's in the text later, lines 61ff)

Reply:
Not gonna hold up a merge review for this bc it's in our docs as-is, but it should really be "run" or "enter" the following command. Based on https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command